### PR TITLE
W-10591849- clarify no_proxy behavior

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

**Review by rohit-verma7 — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/install-manual.adoc [hunk 1]
```diff
@@ -198,7 +198,7 @@ The following variables can be defined based upon your organization's requiremen
 |Parameter | Description | Example
 | `RTF_HTTP_PROXY` | The hostname and port of the HTTP proxy server that will forward outbound HTTP requests. | `+http://1.1.1.1:80+`
 | `RTF_MONITORING_PROXY` | The hostname, port, and optional username and password of the SOCKS5 proxy server that will forward metrics and logs to Anypoint Monitoring. | `socks5://user:pass@1.1.1.2:800`
-| `RTF_NO_PROXY` | A comma-separated list of hosts which should bypass the proxy. | `1.1.1.1,no-proxy.com`
+| `RTF_NO_PROXY` | A comma-separated list of hosts which should bypass the proxy. Runtime Fabric looks for upper-case values for the hosts, but if upper-case values are missing, Runtime Fabric will pick lower-case values. If both values are present, Runtime Fabric picks the upper-case values. | `1.1.1.1,no-proxy.com`
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/install-manual.adoc]: @rohit-verma7 to clarify aren't we talking about reading values from NO_PROXY (uppercase) followed by no_porxy (lowercase) key?

Comment (IsaacEldridge) [reply modules/ROOT/pages/install-manual.adoc]: @kmendiratta-crm do you mean 

Runtime Fabric looks for the key specified as NO_PROXY in upper case but will pick no_proxy in lower case if upper-case values are not used. If both upper-case and lower-case values are present, Runtime Fabric picks the upper-case values.

Comment (kmendiratta-crm) [reply modules/ROOT/pages/install-manual.adoc]: yes that is my understanding @IsaacEldridge.

Comment (rohit-verma7) [reply modules/ROOT/pages/install-manual.adoc]: Yes thanks for clarifying it @kmendiratta-crm , i think i might have misinterpreted it earlier.

**Review by sathya0 — APPROVED** [review]

**Review by kmendiratta-crm — APPROVED** [review]

**Pull request merged**